.readthedocs.yml

```diff
@@ -1,5 +1,17 @@
+# https://docs.readthedocs.io/en/stable/config-file/v2.html
+version: 2
+
+sphinx:
+  fail_on_warning: false
+
+formats:
+  - pdf
+  - epub
```

Comment:
Is epub used?

Reply:
It was enabled in the RTG GUI before, so I left it enabled.